uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/grpc/GRPCControlServer.java

```diff
@@ -274,11 +282,17 @@ public void publishMqtt(MqttPublishRequest request, StreamObserver<MqttPublishRe
             logger.atInfo().log("Publish: connectionId {} topic {} QoS {} retain {}",
                     connectionId, topic, qos, isRetain);
 
+            Map<String, String> userProperties = new HashMap<>();
```

Comment:
Do not initialize userProperties by empty HashMap when no user properties in gRPC request.